source/Core/Smarty/Plugin/function.oxscript.php

```diff
@@ -48,6 +49,7 @@ function smarty_function_oxscript($params, &$smarty)
     $priority = !empty($params['priority']) ? $params['priority'] : 3;
     $widget = !empty($params['widget']) ? $params['widget'] : '';
     $isInWidget = !empty($params['inWidget']) ? $params['inWidget'] : false;
+    $isAsync = (!empty($params['async']) && ( $params['async'] == 'true' || $params['async'] == '1')) ? true : false;
```

Comment:
Maybe this could be simplified to something like: $isAsync = isset($params['async'])?(bool)$params['async']:false; ?